exercises/complex-numbers/src/complex_numbers.h

```diff
@@ -0,0 +1,19 @@
+#ifndef _COMPLEX_NUMBERS_H_
+#define _COMPLEX_NUMBERS_H_
+
+typedef struct complex_t {
```

Comment:
Here you only need `typedef struct { double real; double imag; } complex_t;`, you don't need to write `complex_t` twice.

See [this stackoverflow post](https://stackoverflow.com/a/612350)